en/2-relay-chain/1-launch.md

```diff
@@ -33,15 +33,15 @@ is using the defaults, no other nodes on the relay chain or parachains can use t
 
 To explore and interact with the network, you can use the Polkadot JS Apps UI. If you've started
 this node using the command above, you can access the node as
-https://polkadot.js.org/apps/#/?rpc=ws://localhost:9944
+https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944
```

Comment:
what was the problem with the previous version ?

Reply:
was not working for me, this is the URL that was given when manually setting this.